schema.js

```diff
@@ -188,6 +215,96 @@ const resolvers = {
     },
     matchInfo: async (root, { region, matchId, playerId }, { dataSources }) =>
       getMatchInfo({ dataSources, region, matchId, playerId }),
+    getSeasonStats: async (
+      root,
+      { region, playerId, season },
+      { dataSources }
+    ) => {
+      const information = await dataSources.pubgAPI.getSeason(
+        region,
+        playerId,
+        season
+      );
+
+      const {
+        data: {
+          attributes: { gameModeStats }
+        }
+      } = information;
+
+      const summed = reduce(
+        values(gameModeStats),
+        (
+          accum,
+          {
+            kills,
+            assists,
+            losses,
+            roundsPlayed,
+            wins,
+            top10s,
+            suicides,
+            teamKills,
+            walkDistance,
+            rideDistance,
+            vehicleDestroys,
+            heals,
+            revives,
+            damageDealt,
+            roundMostKills,
+            longestKill,
+            timeSurvived,
+            longestTimeSurvived
+          },
+          idx,
+          arr
+        ) => ({
+          kills: accum.kills ? accum.kills + kills : kills,
+          assists: accum.assists ? accum.assists + assists : assists,
+          losses: accum.losses ? accum.losses + losses : losses,
+          roundsPlayed: accum.roundsPlayed
+            ? accum.roundsPlayed + roundsPlayed
+            : roundsPlayed,
+          wins: accum.wins ? accum.wins + wins : wins,
+          top10s: accum.top10s ? accum.top10s + top10s : top10s,
+          suicides: accum.suicides ? accum.suicides + suicides : suicides,
+          teamKills: accum.teamKills ? accum.teamKills + teamKills : teamKills,
+          kdRatio:
+            idx === arr.length - 1
+              ? (accum.kills + kills) / (accum.losses + losses)
+              : 0,
+          runningDistance: accum.walkDistance
+            ? accum.walkDistance + walkDistance
+            : walkDistance,
+          drivingDistance: accum.rideDistance
+            ? accum.rideDistance + rideDistance
+            : rideDistance,
+          vehiclesDestroyed: accum.vehicleDestroys
+            ? accum.vehicleDestroys + vehicleDestroys
+            : vehicleDestroys,
+          heals: accum.heals ? accum.heals + heals : heals,
```

Comment:
let's also get `boosts` from their API and add them to heals .. so `heals` = `boosts` + `heals`